scripts/set_xtensa_params.sh

```diff
@@ -63,6 +63,11 @@ case "$platform" in
 	XTENSA_CORE="ace10_LX7HiFi4_2022_10"
 	TOOLCHAIN_VER="RI-2022.10-linux"
 	;;
+    ptl)
+	PLATFORM="$platform"
```

Comment:
Could you do everyone a favor and make this the default just before the `case` in first, cleanup commit? Overriding it only when different.

This should get rid of almost 20 lines.

Reply:
Good idea, done now!